### PR TITLE
docs(packages): add missing READMES to packages

### PR DESCRIPTION
**Pull request by ev1stensberg**
**What kind of change does this PR introduce?**
Enhancement

**Did you add tests for your changes?**
N/A
**If relevant, did you update the documentation?**
Yes
**Summary**
Adds README files to all our mono repo packages

**Does this PR introduce a breaking change?**
no

**Other information**
=)

### CODE REVIEW

**Comment by fokusferit** [comment]
But Ci failed? 😅

**Comment by ematipico** [comment]
And not reviewed 🤔

**Comment by ev1stensberg** [comment]
CI is @dhruvdutt s task

File: packages/generate-loader/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+### Node
+
+```js
+const generateLoader = require("@webpack-cli/generate-loader");
+generateLoader();
+```
+
+### CLI (via `webpack-cli`)
+```bash
+npx webpack-cli generate-loader
```
Comment (ematipico) [inline packages/generate-loader/README.md]: We should always split installation and usage of the packages in local and global:
```bash
npx webpack-cli generate-loader
```
and
```bash
webpack-cli generate-loader
````

In this file we mixed, which is confusing for the user

File: packages/generate-plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# webpack-cli generate-plugin
+
+## Description
+
+This package contains the logic to initiate new plugin projects.
```
Comment (ematipico) [inline packages/generate-plugin/README.md]: `initiate` => `create`

File: packages/generators/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# webpack-cli generators
+
+## Description
+
+This package contains all webpack-cli related yeoman generators.
```
Comment (ematipico) [inline packages/generators/README.md]: We should explain what the `generators` do

File: packages/remove/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# webpack-cli remove
+
+## Description
+
+This package contains the logic to remove  properties of a webpack configuration file. It will run a generator that prompts the user for questions of which property to remove in their webpack configuration file.
```
Comment (ematipico) [inline packages/remove/README.md]: `the user some question about which`

**Comment by ematipico** [comment]
@ev1stensberg left some comments about some changes. Unfortunately this is already been merged (don't know why).

**Comment by ev1stensberg** [comment]
Submit a PR with patch label @ematipico

**Pull request merged**